static/src/javascripts-legacy/projects/commercial/modules/hosted/video.js

```diff
@@ -28,13 +27,12 @@ define([
     events,
     videojsOptions,
     fullscreener,
-    contains,
     loadingTmpl
 ) {
     var player;
```

Comment:
possibly out of scope of this PR, but could this `var` be declared within `setupVideo`?